pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -189,6 +190,11 @@ func (m SQSMonitor) retrieveNodeName(instanceID string) (string, error) {
 		},
 	})
 	if err != nil {
+		// the ec2 client doesn't use aws error codes
```

Comment:
this should work:

```
if aerr, ok := err.(awserr.Error); ok ; aerr.Code() == "InvalidInstanceID.NotFound" {
   ...
}
```

Generally InvalidInstanceID.NotFound means the instance just started and isn't describe-able yet. It might be better to do a couple retries rather than just erroring immediately.